client/jetpack-connect/style.scss

```
@@ -41,12 +41,30 @@
 		&[disabled],
 		&:disabled {
 			background: var( --color-white );
-			border-color: var( --color-neutral-50);
+			border-color: var( --color-neutral-50 );
 			color: var( --color-neutral-50 );
 		}
 	}
 }
 
+.jetpack-connect__step {
+	margin: 0 auto;
+	max-width: 600px;
+	text-align: initial;
+
+	input[type='radio']:focus {
+		box-shadow: 0 0 0 2px var( --color-jetpack-light );
+	}
+
+	.site-type__option.is-selected {
+		box-shadow: 0 0 0 1px $green-jetpack inset;
+
+		input[type='radio']:checked::before {
+			background-color: $green-jetpack;
```

I'm a bit curious on this one, is there any reason to avoid using a variable?

```suggestion
			background-color: var( --color-jetpack );
```

Ideally $green-jetpack wouldn't exist anymore but #30468

---

That's a good call 👍 No reason at all 😉 Fixed in https://github.com/Automattic/wp-calypso/pull/30699/commits/b36a86f56a4165cc275007874475e42cc157f3ab